<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitErrorPlotter</class>
 <widget class="QWidget" name="FitErrorPlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QLineEdit" name="register_path">
         <property name="placeholderText">
          <string>register path</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QPushButton" name="run">
         <property name="text">
          <string>Run selected protocols</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="register_browse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="validate">
         <property name="text">
          <string>Validate...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QComboBox" name="validateTarget">
         <item>
          <property name="text">
           <string>Target p.v.</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Median p.v.</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Low-cost p.v.</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="matchedLabel">
         <property name="text">
          <string>Fits matched against register: 0/0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="epoch">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="index">
         <property name="text">
          <string>Index...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pdf">
         <property name="text">
          <string>PDF...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="params">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Target</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Final</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Epoch...</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Boxplot</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Group by</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="protocolMeans">
             <property name="text">
              <string>Protocol means</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="groupCat">
             <item>
              <property name="text">
               <string>Group/Fit</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cell</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="flipCats">
             <property name="text">
              <string>Flip major/minor</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QCheckBox" name="groupMeans">
             <property name="text">
              <string>Group means</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QCheckBox" name="splitTarget">
             <property name="text">
              <string>Split groups by target vs fitted</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="trace_tab">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>Traces</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QRadioButton" name="trace_single">
             <property name="text">
              <string>Single trace</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="trace_stack">
             <property name="text">
              <string>Stacked traces</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="trace_stimidx"/>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="trace_rec">
             <property name="text">
              <string>Recording</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="trace_sim">
             <property name="text">
              <string>Simulation</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="trace_stim">
             <property name="text">
              <string>Stimulation</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="4" rowspan="6">
        <widget class="QTableWidget" name="protocols">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>60</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Avail</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Protocol</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QComboBox" name="validate_protocol">
         <item>
          <property name="text">
           <string>Selected protocols (full obs)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Target protocol (original obs)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Both, and all parameter sets</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCustomPlot" name="plot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
